lib/hammer_cli_katello/content_view_purge.rb

```diff
@@ -0,0 +1,59 @@
+module HammerCLIKatello
+
+  class ContentViewPurge < HammerCLIKatello::Command
```

Comment:
Perhaps this file should be put in `hammer_cli_katello/content_view.rb` with the rest of the content_view hammer commands. Only subcommands get their own file usually (i.e. `hammer_cli_katello/content_view_puppet_module.rb`)